src/common/user-store.ts

```diff
@@ -85,6 +88,23 @@ export class UserStore extends BaseStore<UserStoreModel> {
     return semver.gt(getAppVersion(), this.lastSeenAppVersion);
   }
 
+  @action
+  setTableSizing(clusterId: string, tableId: string, sizes: number[]) {
```

Comment:
Besides `tableId` probably better to keep in format `{[columnId: sting]: number /*size*/}`. Wdyt?